tests/test_jobs.py

```diff
@@ -50,6 +50,7 @@
     'dispatcher_token': 'tokened',
     'hq_url': 'http://example.com/hq',
     'hq_token': 'tokened',
+    'worker_host': 'albus.frigg.io'
```

Comment:
ah yes, but the hostname is not "real" ?  it could be dummy.dummy.com ? (not that it should)